x-pack/platform/plugins/shared/index_management/public/application/sections/home/index_list/index_table/index_table.js

```diff
@@ -123,7 +124,7 @@ const getColumnConfigs = ({
         }),
         order: 60,
         render: (index) => {
-          return Number(index.documents ?? 0).toLocaleString();
+          return <DocCountCell indexName={index.name} httpSetup={http} />;
```

Comment:
another concern we now have 2 competing sources of truth for doc counts. `index.documents` is still used in other parts of the table including sorting. Which can potentially create a count corruption. You might be sorting with one number (`_stats`) while displaying another (`_count`).

Reply:
Yes, we might just remove sorting by document count.

Reply:
well either way on main branch it works as "UX expected"

<img width="1395" height="812" alt="image" src="https://github.com/user-attachments/assets/8d721b17-0aea-4ee4-b554-352c1a5396c8" />

and if we merge this as is now this will introduce a regression

So I don't think we should merge this unless we fix this properly or discuss how to move on with this wdyt?

Reply:
>We can tweak it, but the idea is we can overcome the potential slowdown of _count by doing more incremental async loading of doc counts in chunks

The overall job size is the same no matter how finely its divided.